### DISCUSSION TITLE
Can this library be used with Expo managed workflow?

### DISCUSSION THREAD

**Discussion by zirlman** [hyochan/react-native-iap#2146] [Q&A] [answered]
Hello react-native-iap community,

I'm working on a project that uses Expo managed workflow and we want to add in-app purchases. Looking at Expo's [InAppPurchase](https://docs.expo.dev/versions/latest/sdk/in-app-purchases/) documentation I saw that this library is suggested to be used. That being said I wasn't able to find an answer whether it supports managed workflow. 

My assumption is that it doesn't since the Expo package doesn't support it as well, however, I just wanted to be sure if my assumption is correct.

**Comment by andresesfm** [comment]
@hyochan I think you published the library as an expo plugin, no? Sorry I don't have much context at this time

**Comment by yume-chan** [comment, marked as answer]
I roughly followed the first reply from https://expo.canny.io/feature-requests/p/means-to-set-missingdimensionstrategy and it worked.

**Reply by zirlman** [reply]
Thank you. 🙏
